File: composer/src/main/java/io/piano/android/composer/model/ActiveMeter.java
package io.piano.android.composer.model;

import androidx.annotation.Keep;
import androidx.annotation.NonNull;

import com.google.gson.annotations.SerializedName;

@Keep
public class ActiveMeter {
    @NonNull
    @SerializedName("meterName")
    public String meterName = "";

    @SerializedName("views")
    public int views;

    @SerializedName("viewsLeft")
    public int viewsLeft;

    @SerializedName("maxViews")
    public int maxViews;

    @SerializedName("totalViews")
    public int totalViews;
}

File: composer/src/main/java/io/piano/android/composer/model/ActiveMeter.kt
package io.piano.android.composer.model

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
public class ActiveMeter(
    @JvmField public val meterName: String,
    @JvmField public val views: Int,
    @JvmField public val viewsLeft: Int,
    @JvmField public val maxViews: Int,
    @JvmField public val totalViews: Int,
)
